src/test/run-pass/const-negation.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#[deny(const_err)]
+
+fn main() {
```

Comment:
test for @arielb1 's comment https://github.com/oli-obk/rust/commit/928be878ee9ffb303d3a82106d582bc82ec65acc#commitcomment-15586839